app/static/js/common.js

```diff
@@ -93,12 +107,41 @@ $(document).ready(function() {
             type: 'POST',
             success: (data) =>{
                 APP.toast("You've logged out!")
-                APP.redirect_after('/', 2000);
+                APP.redirect_after('/', 500);
             },
             error: (jqXHR, textStatus, errorThrown) => {
                 console.log("Log out error: " + textStatus);
                 APP.redirect('/');
             }
         });
-    })
-});
\ No newline at end of file
+    });
+
+    $('#goto-team-base').on('click', (event)=>{
+        APP.redirect('/team/');
+    });
+    $('#goto-user-schedule').on('click', (event)=>{
+        APP.redirect('/team/schedule');
+    });
+});
+
+class Week{
+    constructor(year, week){
+        this.moment = moment().startOf('isoWeek').set({'isoWeekYear': year, 'isoWeek': week});
+    }
+    static thisWeek(){
+        return Week.relative(0);
+    }
+    static relative(offset=0){
+        var m = moment().startOf('isoWeek').add(offset, 'w')
+        return new Week(m.get('isoWeekYear'), m.get('isoWeek'));
+    }
+    isCurrent(){
+        return this.moment.isSame(moment().startOf('isoWeek'));
+    }
+    isoWeek(){
+        return this.moment.format('GGGG-[W]WW');
+    }
+    month(){
+        return moment(this.moment).startOf('isoWeek').add(3, 'days').format("MMMM");
+    }
+}
```

Comment:
Time for you to hate me but...
_New line!_
It messes up with tools that read the file and `git blame`
```suggestion
}

```